mantidproject/mantid
# Added images to sans and refletometry release notes

**Description of work.**
This adds images to the reflectometry and sans release notes. 
**Report to:** Nick 

**To test:**
Build docs and check images look fine


Fixes Na. 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
